sagemaker-neo-notebooks/edge/cpp-integration/README.md

```diff
@@ -0,0 +1,219 @@
+
+# C++ Tutorial
+
+## Prerequisite
+
+Please review the following prerequisites tasks before proceed with tutorial
+
+### Environment Setup
+
+#### Environment Setup for Windows
```

Comment:
Prerequisite Software and Environment...?

Reply:
it's revised as suggested.